packages/map/src/map/utils.jsx

```diff
@@ -21,3 +21,36 @@ export const mergeChildrenBounds = (bounds) => {
   if (!isFilledArray(bounds)) return undefined;
   return bounds.flat(1);
 };
+
+export const getMapOptions = ({
```

Comment:
Maybe one test case for this whole thing? (Check if options schema is alright?) Wouldn't test every possible case.